lib/assets.js

```diff
@@ -21,20 +21,16 @@ function asset(options = { inline: [] }) {
 }
 
 
-const loadAssetsMap = () => {
+
+const loadAssetsMap = (origin) => {
   assetsMap = {};
 
-  const entryPath = resolve('app/assets/entry.html');
-  const distDir = 'public/dist';
-  const entryDir = dirname(entryPath);
-  const entryName = basename(entryPath);
-  const resultPath = resolve(distDir, entryName);
-  const entryPaths = getAssetPaths(fs.readFileSync(entryPath, 'utf8'));
-  const resultPaths = getAssetPaths(fs.readFileSync(resultPath, 'utf8'));
+  const entryDir = 'app/assets';
+  const map = JSON.parse(fs.readFileSync(resolve('assetsMap.json'), 'utf8'));
```

Comment:
We probably should change the `entryDir` to where the assetsMap.json file lives and update all the entry file paths in the esbuild script to be relative to that entryDir path